src/constants/config.json

```diff
@@ -1,8 +1,62 @@
 {
-  "cartoDB": {
-    "mapVisualisationUrl": "https://shaunanoordin-zooniverse.cartodb.com/api/v2/viz/e04c2e20-a8a9-11e5-8d6b-0e674067d321/viz.json",
-    "dataTable": "wildcam_gorongosa_cameras_201601",
-    "dataLayerIndex": 1
+  "mapExplorer": {
```

Comment:
I can extract the mapExplorer.species into a separate... let's say... selectorOptions.json file, since filter options like 'species' will only grow as the project evolves, yes, separating it from config.json makes sense. However, at that stage, I wonder if it'd make more sense to extract all Map-related configs and constants (including mapExplorer.cartoDB, mapExplorer.baseLayers, mapExplorer.mapCentre, and the ever-growing list of filter options) into a component-centric logical unit called mapExplorer.config.json. Functionally the same, just a team coding style issue - thoughts?